### ISSUE TITLE
Character Sprites 'Console' config bug

### ISSUE THREAD

**Issue by FScaccheri** [higurashi-mod/watanagashi#94]
**_TL;DR: dialogue doesn't advance if the console sprites are on (remake sprites work fine). Quick fix is replacing remake sprites with console sprites in the game [files._**] 

I am experiencing a strange bug with the console sprites in the Watanagashi chapter, with the manual instalation of the patch. I get the same bug if I use the windows installer, but I don't get this bug in other chapters (at least not with chapter 1).
I'm linking a video that I recorded, along with some explanation of what's going on:

- [Video of the bug](https://youtu.be/JrBKP_YtEjg)

I start by showing the manual installation (just to make sure I did everything correctly). You can skip to [3:45](https://youtu.be/JrBKP_YtEjg?t=225), when the game opens.
After I set the **Character Sprites** setting to **Console** I open my save file and attemp to continue the game. As you can see in th video, the first line of dialogue comes up fine, but when I try to go to the next line (by clicking or with the keyboard) the arrow pointing to the right dissapears and the text that should come up simply doesn't. I try to press some buttons to no avail and I close the game. 
Then, at [5:02](https://youtu.be/JrBKP_YtEjg?t=302), I set the sprites setting to **Remake**, to show that the remake sprites do work fine, but the bug appears again if I change it back to **Console**. Also, at [5:54](https://youtu.be/JrBKP_YtEjg?t=354), if I try to exit with **Exit to Menu**, the sprites, text box and music are gone, but I'm now stuck with the background, and I have to exit the game externally (Alt + F4).

I have found some kind of fix though.
If I go to the _StreamingAssets_ folder of the game and copy the _portrait_ and _sprite_ folders from _CG_ into the folder _CGAlt_ (overwriting the Remake sprites with the Console sprites) I can play the game just fine with the Console sprites by selecting the **Remake** setting in **Character Sprites** (shown to work fine at [7:11](https://youtu.be/JrBKP_YtEjg?t=431) in the video).

**Comment by enumag** [comment]
Is the save file you're using from an older version of our patch by any chance?

**Comment by FScaccheri** [comment]
I'm not 100% sure, but I'd still dare to say it actually is a save from an older version, since the bug appeared after a fresh install of windows (and thus a fresh install of the patch), but on an older save.

**Comment by enumag** [comment]
Older saves are expected to not work. Can you try to reproduce it by not using any save file but the chapter select menu instead?

**Comment by drojf** [comment]
Maybe it's worth adding a note on the wiki about saves? I don't see any information about saves on the Higurashi wiki...

**Comment by FScaccheri** [comment]
> 
> 
> Older saves are expected to not work. Can you try to reproduce it by not using any save file but the chapter select menu instead?

I just tried this with the Console sprites option on and it worked just fine! Thanks a lot for the help and fast response

**Comment by enumag** [comment]
@drojf Yeah... not sure where to add it though. They don't break always. Any ideas @ItaloKnox?

**Comment by ItaloKnox** [comment]
Said and done. There's a warning at the intro page now.
![firefox_2019-02-26_15-33-25](https://user-images.githubusercontent.com/4702556/53437291-e7239280-39db-11e9-9add-cd1c878400cc.png)

**Comment by enumag** [comment]
@ItaloKnox Awesome!
